### DISCUSSION TITLE
Time Tracking

### DISCUSSION THREAD

**Discussion by timvahlbrock** [Oeffis/Oeffis#2] [General]
We need a way to track the time we spent on certain tasks, ideally one that integrates well with GitHub.

**Comment by timvahlbrock** [comment]
GitHub Issues have an existing field called "size" that has 5 values from tiny to X-Large. These don't seem to be too granular, but they might be sufficient for our purposes. We could define certain limits for which amount of time corresponds to which level of size, but in my opinion the amount of time spend on an issue can only be determined to a certain level of preciseness anyway - so the levels might be sufficient.

**Comment by JaJako** [comment]
There are some exciting tools that integrate with GitHub via browser extension. Next to each issue a start/stop button is being displayed (or you can edit the time invested manually). But most of them aren´t free to use. "Clockify" (https://clockify.me/) for example seems to have a free plan. 
But for our use case most of those tools seems to be overloaded with additional (management) features.

**Comment by JaJako** [comment]
This GitHub App/Plugin looks interesting: https://github.com/marketplace/7pace-for-github . You get timetracking per issue build into GitHub. And during beta the GitHub integration is free.

**Reply by timvahlbrock** [reply]
Did you see any info how long the beta phase will last?

**Reply by JaJako** [reply]
So far I haven´t found some kind of "end date". The changelog goes back until 2020-12 - so it has been in beta for a while, I guess.

**Reply by JaJako** [reply]
"Free for personal use" is not matching for our case, right?

**Reply by timvahlbrock** [reply]
I think that depends on the concrete terms, "personal use" probably isn't a protected term. But my first guess would also be "no".

**Comment by JaJako** [comment]
For the bachelor project we used "Monday.com" - it's free for students. It also integrates with Github (we haven´t used that integration back then): https://monday.com/integrations/github

**Comment by Paulomart** [comment]
Using a simple Excel Sheet could also be possible :)

**Comment by timvahlbrock** [comment]
For now closed with solution "Specific format on issue"
